components/Table/Table.js

```diff
@@ -10,21 +10,39 @@ export class Table {
   }
 
   createTable() {
-    this.thead = document.createElement("thead");
+    this.thead = document.createElement("div");
+    this.thead.classList.add("table-header");
     this.tableContainer.append(this.thead);
 
-    this.tbody = document.createElement("tbody");
+    this.tbody = document.createElement("div");
+    this.tbody.classList.add("table-body");
     this.tableContainer.append(this.tbody);
 
     this.createHeader();
   }
 
   createHeader() {
-    const tr = document.createElement("tr");
+    const tr = document.createElement("div");
+    tr.classList.add("table-header-row");
 
     const cells = this.columnsConfig.map((col) => {
-      const cell = document.createElement("th");
+      const cell = document.createElement("div");
+      cell.classList.add("table-header__cell");
+
+      const btnContainer = document.createElement("div");
```

Comment:
Since we do not have a single button there, let's rename this variable
`btnContainer` -> `btnsContainer`
I would call it `actionsContainer`